src/datagrids/style.css

```diff
@@ -0,0 +1,166 @@
+.ag-theme-alpine .ag-icon-none{
+  --ag-icon-font-family: 'Material Design Icons';
+  --ag-icon-font-code-none: '\F0A65';
+}
+.ag-theme-alpine .ag-icon-desc{
+  --ag-icon-font-family: 'Material Design Icons';
+  --ag-icon-font-code-desc: '\F06B6';
+  padding-top: 6px;
+  color: dodgerblue;
+}
+.ag-theme-alpine .ag-icon-asc{
+  --ag-icon-font-family: 'Material Design Icons';
+  --ag-icon-font-code-asc: '\F06B7';
+  padding-bottom: 6px;
+  color: dodgerblue;
+}
+.ag-theme-alpine {
+  --ag-header-height: 30px;
+}
+.ag-theme-alpine .ag-header-cell {
+  color: #6d6868;
+  font-weight: 600;
+  font-size: 12px;
+}
+ 
+.ag-theme-alpine #page-size {
+  border-width : 0px;
+  font-size: small;
+  padding-right: 20px;
+  padding-top: 4px;
+  padding-bottom: 3px;
+  padding-left: 3px;
+  /* padding: 0 1.5em 0 1em; */
+  /* margin: 0 0 0 1em; */
+}
+.ag-theme-alpine .ag-header-row-column {
+  /* font-size: smaller; */
+  /* font-weight: bold; */
+  color: #aaa;
+}
+.index-header {
+  margin-bottom: 24px;
+  margin-bottom: 24px;
+  font-weight: 800;
+  font-size: 18px;
+}
+.ag-theme-alpine .grid-header {
+  border: 1px solid;
+  border-color: #aaa;
+  border-bottom: none; 
+  padding-left: 18px;
+  padding-top: 2px;
+  padding-bottom: 2px; 
+  color: #6d6868; 
+  font-weight: bolder
+}
+.ag-theme-alpine .grid-bottom {
+  float: right;
+  width: 100%; 
+  padding: 10px;
+  display: flex; 
+  justify-content: end; 
```

Comment:
Let's take Vite's advice here if we can:
![Screen Shot 2023-02-17 at 5 27 38 PM](https://user-images.githubusercontent.com/4667097/219824304-64aca567-73e6-4356-9d53-ffa4c5a1e3c3.png)